packages/picasso/src/TagRectangular/index.ts

```diff
@@ -1,6 +1,7 @@
 import { OmitInternalProps } from '@toptal/picasso-shared'
 
-import { Props } from './types'
+import { Props, VariantType } from './types'
 
 export { default } from './TagRectangular'
 export type TagRectangularProps = OmitInternalProps<Props>
+export type TypeRectangularVariantType = VariantType
```

Comment:
```suggestion
export type TagRectangularVariantType = VariantType
```
It is confusing to be to have the name which both starts and ends with `Type` :)